### ISSUE TITLE
Responsible region and district of an event are used for location

### ISSUE THREAD

**Issue by bernardsilenou** [hzi-braunschweig/SORMAS-Project#4988]
### Bug Description
- The region and district initially used to define the responsible jurisdiction are now used to represent the location of the event. 
- Event can be created without a responsible region and district. In addition, region and district are requited when country is NULL
- ### Steps to Reproduce
1. Create an event with a country that is not the same as that on server
2. It is possible to save event

### Expected Behavior
- Add a responsible region and district to an event and make it a required variable
- Use this responsible jurisdiction to validate which  users should have full access to the event
- When country is NULL, then no need to show region and district as required 
- Chen country is NULL or not the same as country on server,  region + district + community should be grayed out and non reactive.
### Screenshots
![image](https://user-images.githubusercontent.com/12996724/113362035-af6f3d80-934d-11eb-9e34-4d92cbab1533.png)

### System Details
* Device:
* SORMAS version:
* Android version/Browser:
* Server URL:
* User Role:

### Additional Information
![image](https://user-images.githubusercontent.com/12996724/113362684-31139b00-934f-11eb-9b02-0dfd0e31add5.png)

**Comment by kwa20** [comment]
@bernardsilenou Is the server locale set in your case? If so, setting the country to the same as the locale should make region and district mandatory. If another country is selected, the fields become optional and the event participants responsible region and district become mandatory instead. This is expected behaviour. 

<img width="664" alt="Screen Shot 2021-04-02 at 12 46 23 PM" src="https://user-images.githubusercontent.com/70317594/113409781-508fdf80-93b2-11eb-9e60-f0f82e0e0806.png">

A problem I could think of would be if the locale is not set but neither event or event participant region and district are mandatory.

**Comment by bernardsilenou** [comment]
@kwa20 The local was set on the system that i did the test.
- The point I am raising is about the event, not ep. Even if the country local is set and you create an event, then any user connected to the system can modify the event meta data.
- For every entity in the system,  we need to have only one responsible jurisdiction to have full access at a point in time. If country of the event is not the same as that on server, then this is missing. Is that tight?
- My suggestion is to change this statement: ` If another country is selected, the fields become optional and the event participants responsible region and district become mandatory instead. This is expected behaviour.` by always having a responsible region and district to the event even if the country is not the same as the local country. What to you think pls?

**Comment by kwa20** [comment]
@bernardsilenou I don't think it should be mandatory to set a responsible region and district for a different country than yours because you might not necessarily know which region and district is responsible. If events are documented that happened in other countries, the main thing users have to worry about are persons that were participants of these events and all users should have access to the events anyway. I think it would then be more important to assign participants to responsible regions and districts inside your own country.

**Comment by bernardsilenou** [comment]
@kwa20 
- The responsible region and district is always in the the country of the server. 
If an event tool place in France, and is created on a DE server, then the location of event  == France but the responsible jurisdiction is a region in  DE.
- This is the same for event participant, the responsible region and a location can be in different countries
- In addition, ep has event as their parent entity, so any action applies to event would affect ep, eg if a user change the date of event, then all followup of ep are modified.
- I think we should instead always enforce the responsible jurisdiction of the event, the location can be optional and any where in the world.
- Unlike contact, ep can not exist in the system without an event, easier for the user not to document the jurisdiction of event for every ep documented by not making the jurisdiction required.
- I just realized that even the jurisdiction is required for ep, you can still create an ep without providing this data.
- Maybe we discuss this tomorrow? The only issue I can see here is that the event meta data can be modified by every user.

**Comment by MateStrysewske** [comment]
@bernardsilenou @kwa20 This is definitely not a bug. Before the changes we did in the last update, the region and district were simply not editable (and not mandatory) when a different country than the server country was selected. Also please be aware that the server locale has nothing to do with this, it's the country name that has to be filled in.

I think you should indeed discuss how this should be handled again and then come up with a proposed solution.

**Comment by bernardsilenou** [comment]
@MateStrysewske Who are the people to have this discussion pls? Is HZI  to do this refinement?

**Comment by MateStrysewske** [comment]
@bernardsilenou Yes.
